source/Calamari.Aws.Tests/S3Fixture.cs

```diff
@@ -179,7 +179,7 @@ public void UploadPackageWithMetadata()
                 }
 
                 foreach (var userMetadata in userDefinedMetadata)
-                    metadata["x-amz-meta-" + userMetadata.Key.ToLowerInvariant()]
+                    metadata["x-amz-meta-" + userMetadata.Key.ToLower()]
```

Comment:
`ToLower()` uses current culture. What we really want is an extension method to do a case-insensitive assert.

Let's keep `ToLowerInvariant()`, it would be better than `ToLower()` here.